ext/node/polyfills/process.ts

```diff
@@ -52,15 +54,42 @@ export let platform = "";
 // TODO(kt3k): This should be set at start up time
 export let pid = 0;
 
-// TODO(kt3k): Give better types to stdio objects
-// deno-lint-ignore no-explicit-any
-let stderr = null as any;
-// deno-lint-ignore no-explicit-any
-let stdin = null as any;
-// deno-lint-ignore no-explicit-any
-let stdout = null as any;
+// We want streams to be as lazy as possible, but we cannot export a getter in a module. To
+// work around this we make these proxies that eagerly instantiate the underlying object on
+// first access of any property/method.
+function makeLazyStream<T>(objectFactory: () => T): T {
```

Comment:
This won't be free, but it's a tradeoff of startup time vs performance of:

```
import { stdin } from 'node:process';
```

What we should do in the future is make stdin/out/err concrete objects that lazily open their underlying streams.